packages/composites/dxdao/README.md

```diff
@@ -0,0 +1,82 @@
+# Chainlink External Adapter for Dxdao
+
+### Environment Variables
+
+| Required? |            Name            |               Description                |       Options       | Defaults to |
+| :-------: | :------------------------: | :--------------------------------------: | :-----------------: | :---------: |
+|    ✅     | `RPC_URL`  |   The RPC URL to connect to the XDai chain    |  |             |
+
+This adapter also relies on the Token Allocation adapter to fetch the USD price of ETH.  This requires setting up the environment variables 
+for the token allocation adapter as well.  The instructions for that can be found here https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/composites/token-allocation.
+
+
+---
+
+### Input Parameters
+
+No input parameters required
+
+---
+
+## Dxdao Endpoint
+
+This endpoint fetches the TVL(Total Value Locked) inside a pair that is deployed on the XDai chain.  The TVL is returned in USD.
+
+### Input Params
+
+| Required? |            Name            |               Description                |       Options       | Defaults to |
+| :-------: | :------------------------: | :--------------------------------------: | :-----------------: | :---------: |
+|    ✅     | `wethContractAddress`  |   The WETH contract address on the XDai Chain    |  |             |
+|    ✅     | `pairContractAddress` | The pair contract's address on the XDai Chain |   |             |
+|         | `tokenAllocationSource` | Which source to fetch the Ethereum USD price from.  Including this will return the TVL in USD |   |             |
```

Comment:
Instead of giving this it's own param name, we should just pass on the entire input to TA and refer to the TA readme here: https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/composites/token-allocation#readme

Reply:
Wouldn't we have to still construct the input in the code as we are passing in the Ethereum balance at run time?